README.md

```diff
@@ -13,6 +13,7 @@ Role Variables
 --------------
 
 - `nginx_keep_default_configs`: If `true` keep the default site configuration files in `nginx/conf.d`, default `false` (disable them)
+- `nginx_version`: The packaged version of Nginx, optional
```

Comment:
Is this logic dependent of the value of `nginx_stable_repo`?

Reply:
No, you can only install versions that are available in the repo.

Reply:
:+1: maybe we want to group both option together and indicate the relationship?

Reply:
updated